dtfs-central-api/src/v1/controllers/tfm/deal/tfm-get-deal.controller.js

```diff
@@ -13,6 +13,7 @@ const findOneDeal = async (_id, callback) => {
 
   const deal = await dealsCollection.findOne({ _id: { $eq: ObjectId(_id) } });
   let returnDeal = deal;
+  let facilityIds;
 
   if (deal) {
     if (deal.dealSnapshot.dealType && deal.dealSnapshot.dealType === DEALS.DEAL_TYPE.GEF) {
```

Comment:
should this not have optional chaining as well?

Reply:
Good spot.